driver/src/main.rs

```diff
@@ -60,7 +60,7 @@ struct Options {
     /// solver; 'MIP' for mixed integer programming solver; 'NLP' for non-linear
     /// programming solver.
     #[structopt(long, env = "SOLVER_TYPE", default_value = "NaiveSolver")]
-    solver_type: SolverType,
+    solver_type: String,
```

Comment:
nit: the other options are spelled fallback-solver & standard-solver, so maybe naive-solver?